tests/expectations/snapshots/snap_test_expectation_atomic_renderers.py

```diff
@@ -919,11 +909,11 @@
                 "schema": {"type": "array"},
                 "value": ["my_first_col", "my_second_col", "my_third_col"],
             },
-            "condition_parser": {"schema": {"type": "string"}, "value": None},
-            "ignore_row_if": {"schema": {"type": "string"}, "value": None},
+            "column_list_0": {"schema": {"type": "string"}, "value": "my_first_col"},
+            "column_list_1": {"schema": {"type": "string"}, "value": "my_second_col"},
+            "column_list_2": {"schema": {"type": "string"}, "value": "my_third_col"},
```

Comment:
These were missing due to a bug. You will noticed they are referenced in the `template` string on line 919, but the param substitutions are missing from the payload.